mobile/lib/shared/ui/immich_image.dart

```diff
@@ -19,8 +18,7 @@ class ImmichImage extends StatelessWidget {
     this.fit = BoxFit.cover,
     this.useGrayBoxPlaceholder = false,
     this.useProgressIndicator = false,
-    this.type = api.ThumbnailFormat.WEBP,
-    this.preferredLocalAssetSize = 250,
+    this.isThumbnail = false,
```

Comment:
Can we make `isThumbnail` true by default and use a factory `ImmichImage.original` instead to fetch the original image?`ImmichImage` widget is used in few more places for Album thumbnails. The previous implementation also had WEBP by default.

Reply:
yeah, we use the thumbnails more frequently.

Reply:
I would like to make a thumbnail widget separate from this in the future, so I'd prefer to keep the thumbnail constructor the factory and refactor from that later.